include/Gaffer/MetadataAlgo.h

```diff
@@ -111,6 +111,15 @@ GAFFER_API void setBookmarked( Node *node, bool bookmarked, bool persistent = tr
 GAFFER_API bool getBookmarked( const Node *node );
 GAFFER_API void bookmarks( const Node *node, std::vector<NodePtr> &bookmarks );
 
+/// Viewer Hotkeys
+/// ==============
+///
+/// \todo doc
+
+GAFFER_API void setViewerHotkey( Node *node, int hotkey );
```

Comment:
I've renamed the methods and I've added a method for removal. I've opted for that because I've also changed some code to allow '0' as a valid bookmark. Doing so adds just one more "slot" for people to use.